app/src/main/java/com/edu/movie/screen/base/HeadFragment.kt

```diff
@@ -0,0 +1,60 @@
+package com.edu.movie.screen.base
+
+import android.content.Context
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Toast
+import com.edu.movie.R
+import kotlinx.android.synthetic.main.fragment_head.*
+
+class HeadFragment private constructor() : Fragment() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+    }
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        return inflater.inflate(R.layout.fragment_head, container, false)
+    }
+
+    override fun onActivityCreated(savedInstanceState: Bundle?) {
+        super.onActivityCreated(savedInstanceState)
+        bottomNavigation.setOnNavigationItemSelectedListener {
+            when (it.itemId) {
+                R.id.homePage -> {
+                    textHeader.text = MOVIES
+                    true
+                }
+                R.id.trendingPage -> {
+                    textHeader.text = TRENDING
+                    true
+                }
+                R.id.genresPage -> {
+                    textHeader.text = GENRES
+                    true
+                }
+                R.id.favoritePage -> {
+                    textHeader.text = FAVORITE
+                    true
+                }
+                else -> false
+            }
+        }
+    }
+
+    companion object {
+        val instance: HeadFragment by lazy {
+            HeadFragment()
+        }
```

Comment:
instane của Fragment không phải là singleton, ông áp dụng theo conventions 2.11 nhé 
https://github.com/framgia/coding-standards/blob/master/eng/android/codingstyleguide.md#211-argument-in-activity-and-fragment